docs/_templates/layout.html

```diff
@@ -1,6 +1,6 @@
 {% extends "!layout.html" %}
 
-{% set css_files = css_files + ['_static/custom.css'] %}
+{% set css_files = css_files + ['_static/css/custom.css'] %}
```

Comment:
this change breaks the css for me. why do you need it?

Reply:
you you're right you actually dont need it - reverting

Reply:
@tmbo done.